Source/FolioReaderKit.swift

```diff
@@ -23,6 +23,10 @@ internal let kCurrentTOCMenu = "com.folioreader.kCurrentTOCMenu"
 internal let kHighlightRange = 30
 internal let kReuseCellIdentifier = "com.folioreader.Cell.ReuseIdentifier"
 
+public extension Notification.Name {
+    static let folioReaderPresentationComplete = Notification.Name("folioReaderPresentationComplete")
```

Comment:
i think `folioReaderPresentationCompleted` is better than `folioReaderPresentationComplete`

Reply:
name changed to folioReaderPresentationCompleted